darshan-util/pydarshan/darshan/lib/accum.py

```diff
@@ -0,0 +1,64 @@
+import math
+
+import darshan
+from darshan.backend.cffi_backend import log_get_derived_metrics
+from darshan.experimental.plots import plot_common_access_table
+
+darshan.enable_experimental()
+
+import numpy as np
+import pandas as pd
+import humanize
+
+
+def log_file_count_summary_table(log_path: str,
+                                 module: str):
+    # the darshan_file_category enum is not really
+    # exposed in CFFI/Python layer, so we effectively
+    # re-export the content indices we need here
+    # so that we can properly index the C-level data
+    darshan_file_category = {"total opened":0,
```

Comment:
Whoops actually, the suggested change her still stands to make this "total files" rather than "total opened" since technically the count will include things that may not have literally been opened.

Reply:
Fixed via [c61a024](https://github.com/darshan-hpc/darshan/pull/868/commits/c61a024b90547163477936397918689477aeeae3)